ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/converters/RequestConverter.java

```diff
@@ -189,4 +200,35 @@ public static String getResponseType(final ImmutableList<Ds3ResponseCode> respon
 
         return builder.build();
     }
+
+    public static ImmutableMap<String, Parameter> buildRequestPayloadMap() {
+        final ImmutableMap.Builder<String, Parameter> requestPayloadMap =  ImmutableMap.builder();
+        requestPayloadMap.put("CreateGetJobRequestHandler",
+                new Parameter(
+                        ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("CreatePutJobRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("CreateVerifyJobRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("EjectStorageDomainBlobsRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("EjectStorageDomainRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("GetPhysicalPlacementForObjectsRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("VerifyPhysicalPlacementForObjectsRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+        requestPayloadMap.put("VerifyPhysicalPlacementForObjectsWithFullDetailsRequestHandler",
+                new Parameter(ParameterModifier.CONST, "ds3_bulk_object_list", "object_list", ParameterPointerType.SINGLE_POINTER, true));
+
+        /*
+        requestPayloadMap.put("CompleteMultiPartUploadRequestHandler",                          "CompleteMultipartUpload");
```

Comment:
I suppose I could uncomment it, there's nothing to prevent it from working as its not being used or tested anywhere yet